packages/quantic/force-app/solutionExamples/main/lwc/generateAnswerButton/generateAnswerButton.js

```diff
@@ -71,6 +71,7 @@ export default class GenerateAnswerButton extends LightningElement {
    * Handles the button click event and triggers answer generation.
    */
   generateAnswer() {
+    this.engine.dispatch(this.actions.setAnswerGenerationMode('manual'));
```

Comment:
Are we reseting the `generated-answer-state` every time there's a new query? Because otherwise we'll be stuck in manual after we have run a manual query once.

Reply:
Indeed it's reset on every query!